stubs/Pillow/PIL/ImageDraw.pyi

```diff
@@ -0,0 +1,148 @@
+        direction: Literal["rtl", "ltr", "ttb"] | None = ...,
+        features=...,
+        language: str | None = ...,
+        stroke_width: float = ...,
+        stroke_fill: _Ink | None = ...,
+        embedded_color: bool = ...,
+        *args,
+        **kwargs,
+    ) -> None: ...
+    def multiline_text(
+        self,
+        xy: tuple[float, float],
+        text: str | bytes,
+        fill: _Ink | None = ...,
+        font: _Font | None = ...,
+        anchor=...,
+        spacing: float = ...,
+        align: Literal["left", "center", "right"] = ...,
+        direction: Literal["rtl", "ltr", "ttb"] = ...,
+        features=...,
+        language: str = ...,
+        stroke_width: float = ...,
+        stroke_fill: _Ink | None = ...,
+        embedded_color: bool = ...,
+    ) -> None: ...
+    def textsize(
+        self,
+        text: str | bytes,
+        font: _Font | None = ...,
+        spacing: float = ...,
+        direction: Literal["rtl", "ltr", "ttb"] | None = ...,
+        features=...,
+        language: str | None = ...,
+        stroke_width: float = ...,
+    ) -> tuple[int, int]: ...
+    def multiline_textsize(
+        self,
+        text: str | bytes,
+        font: _Font | None = ...,
+        spacing: float = ...,
+        direction: Literal["rtl", "ltr", "ttb"] | None = ...,
+        features=...,
+        language: str | None = ...,
+        stroke_width: float = ...,
+    ) -> tuple[int, int]: ...
+    def textlength(
+        self,
+        text: str | bytes,
+        font: _Font | None = ...,
+        direction: Literal["rtl", "ltr", "ttb"] | None = ...,
+        features=...,
+        language: str | None = ...,
+        embedded_color: bool = ...,
+    ) -> int: ...
+    def textbbox(
+        self,
+        xy,
+        text: str | bytes,
+        font: _Font | None = ...,
+        anchor=...,
+        spacing: float = ...,
+        align: Literal["left", "center", "right"] = ...,
+        direction: Literal["rtl", "ltr", "ttb"] = ...,
+        features=...,
+        language: str | None = ...,
+        stroke_width: float = ...,
+        embedded_color: bool = ...,
+    ) -> tuple[int, int, int, int]: ...
+    def multiline_textbbox(
+        self,
+        xy,
+        text: str | bytes,
+        font: _Font | None = ...,
+        anchor=...,
+        spacing: float = ...,
+        align: Literal["left", "center", "right"] = ...,
+        direction: Literal["rtl", "ltr", "ttb"] = ...,
+        features=...,
+        language: str | None = ...,
+        stroke_width: float = ...,
+        embedded_color: bool = ...,
+    ) -> tuple[int, int, int, int]: ...
+
+def Draw(im: Image, mode: str | None = ...) -> ImageDraw: ...
+
+Outline: Any
+
+@overload
+def getdraw(im: None = ..., hints: Container[Literal["nicest"]] | None = ...) -> tuple[None, Any]: ...
+@overload
+def getdraw(im: Image, hints: Container[Literal["nicest"]] | None = ...) -> tuple[Image, Any]: ...
```

Comment:
Are you sure `tuple[Foo, Bar]` is not affected by the mypy bug?

Reply:
It's only `tuple[X, ...]` and `tuple` (as well as others) in type aliases.